automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioMainPage.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.workbench.leonardo.rstudio
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.leonardo.notebooks.JupyterPage
+import org.openqa.selenium.WebDriver
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.duration._
+import scala.util.Try
+
+class RStudioMainPage(override val url: String)(implicit override val authToken: AuthToken,
+                                                implicit override val webDriver: WebDriver)
+    extends JupyterPage
```

Comment:
Just wondering if we should rename `JupyterPage` to `ToolPage` or something if it's meant to be abstract